invoice/templates/invoice/invoice_list.html

```diff
@@ -22,8 +22,8 @@
                 <td>{{ invoice.date }}</td>
                 <td>{{ invoice.vendor.name }} ({{ invoice.vendor.company_name }})</td>
                 <td>{{ invoice.customer.first_name }} {{ invoice.customer.last_name }}</td>
-                <td>{{ invoice.net_total }}</td>
-                <td>{{ invoice.total }}</td>
+                <td>{{ invoice.net_total|floatformat:2 }}</td>
+                <td>{{ invoice.total|floatformat:2 }}</td>
```

Comment:
is this floatformat working?
if it does you wouldnt need the explicit round on the property getter

Reply:
Yes it does work here, but not in the PDF generation